src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -1509,7 +1509,6 @@ private bool SaveResolvedDocument([@CanBeNull]Document resolvedDoc, [@NotNull]Do
 
             // mergedBody:
             FLSliceResult mergedBody = (FLSliceResult) FLSlice.Null;
-            if (resolvedDoc != null)
             if (!ReferenceEquals(resolvedDoc, remoteDoc)) {
```

Comment:
Not sure why `if (resolvedDoc != null)` is added here.. Remove.

Reply:
Looks like a merge error to me